### PR TITLE
Fix problems while using local installed packages

### PR DESCRIPTION
**Pull request by ClausKlein**
update CPM.cmake version too

This should fix this issue https://github.com/TheLartians/ModernCppStarter/issues/176

### CODE REVIEW

**Comment by ClausKlein** [comment]
@TheLartians Would you like to review this MR?

File: standalone/CMakeLists.txt [hunk 1]
```diff
@@ -12,11 +12,15 @@ include(../cmake/CPM.cmake)
...
+  VERSION 3.1.1
   OPTIONS "CXXOPTS_BUILD_EXAMPLES NO" "CXXOPTS_BUILD_TESTS NO" "CXXOPTS_ENABLE_INSTALL YES"
 )
 
-CPMAddPackage(NAME Greeter SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/..)
+CPMAddPackage(
+  NAME Greeter
+  SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/..
+  FORCE ON
+)
```
Comment (TheLartians) [inline standalone/CMakeLists.txt:23]: Can you shortly explain why `FORCE ON` is needed here?
If the Greeter project has already been added before I would expect it to use the previously added version?

**Review by TheLartians — COMMENTED** [review]
If I understand correctly, these changes are required when using this package with `CPM_USE_LOCAL_PACKAGES` defined, right?
 
Tbh I'm conflicted on these changes as it adds some complexity to the starter for a rather niche use case. IMO instead it could make sense mentioning in the Readme that the starter does not support this mode out of the box as it isn't recommended for CPM anyways as it has known issues.

File: standalone/CMakeLists.txt (hunk 1 above)
Comment (TheLartians) [reply standalone/CMakeLists.txt:23]: I think I understand, see review comment above.

**Pull request closed without merging**